openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -147,10 +147,6 @@ def get(self, request, username):
         """
         try:
             account_settings = get_account_settings(request.user, username, view=request.QUERY_PARAMS.get('view'))
-            # Account for possibly relative URLs.
-            for key, value in account_settings['profile_image'].items():
```

Comment:
Ah, now I see what you were saying about relative URLs. It is great to not have this ugly hack anymore!